Radzen.Blazor/Rendering/DialogContainer.razor

```diff
@@ -31,7 +31,7 @@
             }
             else
             {
-                <div class="rz-dialog-titlebar">
```

Comment:
What happened with `rz-dialog-titlebar`?

Reply:
Sorry for inattention, now this should be right